examples/imagenet/input_pipeline.py

```diff
@@ -57,13 +56,21 @@ def distorted_bounding_box_crop(image_bytes,
     max_attempts: An optional `int`. Number of attempts at generating a cropped
         region of the image of the specified constraints. After `max_attempts`
         failures, return the entire image.
+    rng: RNG used for generating seeds for reproducibility of
```

Comment:
Instead of this, add a comment that we need to resolve reproducibility issues, linking to a GitHub issue with details

Reply:
Created the issue and mentioned it in the README file.